students/paul_jurek/lesson03/mailroom.py

```diff
@@ -0,0 +1,126 @@
+"""runs the command line app for the mailroom as part of lesson 3"""
+
+# setup initial donor list
+donors = {'Bill Gates': [100000.00, 5.00, 3000000.00],
+          'Paul Allen': [10.00, 1000000.00],
+          'Warren Buffet': [300000000.00],
+          }
```

Comment:
@natasha-aleksandrova In general, when storing these types of references are dicts preferred over lists?

Reply:
in this particular case, yes dictionary makes the  most sense since donor name is our primary key and key looks are very efficient.

the reason I mentioned a list is because that is what was expected in lesson 3 to get practice with lists **and** in real scenarios you would most likely get a list of records (like a list of dictionaries) from SQL database query.